example/App.js

```diff
@@ -42,6 +42,17 @@ class App extends PureComponent {
     exp.forEach(e => console.log(e));
   };
 
+  getPlacement = async () => {
+    const placement = await QubitSDK.getPlacement(
+        "tsOujouCSSKJGSCMUsmQRw",
+        "LIVE",
+        null,
+        "1ybrhki9RvKWpA-9veLQSg",
+        null
+    );
+    console.log(placement);
+  };
+
```

Comment:
Calling impression or clickthrougn callback is missing in this example app.

Reply:
Added. Once placement object is returned from `NativeModules.QubitSDK.getPlacement()` function, then I call `impression()` method on it.

Reply:
But clickthrough is still missing.
I'm afraid, clickthrough function was never tested?
Don't you think, we should have buttons to send impression and clickthrough, like it is done in Android Test app?

Reply:
We don't have it in case of experiences, but you're right - added two more options.

Actually it hasn't been working correctly as `CallbackRequestTracker` service wasn't started. Extended `QubitSDK` to return this (already started) service so it will be used here, similarly to `EventTracker`